include/Beman/Optional26/detail/iterator.hpp

```diff
@@ -59,6 +63,11 @@ struct contiguous_iterator : public base_contiguous_iterator<T, Container> {
     T* m_current;
 };
 
+// Check concepts for the contiguous_iterator class.
```

Comment:
Move to a test. It's small in the scheme of things, but we don't need to check this every time this header is used.

Reply:
We already have extensive tests for these in `src/Beman/Optional26/tests/detail/iterator.t.cpp`:

```shell
    // non-const iterators
    test(beman::optional::detail::contiguous_iterator<int, container>{});
    test(beman::optional::detail::contiguous_iterator<empty, container>{});
    test(beman::optional::detail::contiguous_iterator<no_default_ctor, container>{});
    test(beman::optional::detail::contiguous_iterator<base, container>{});
    test(beman::optional::detail::contiguous_iterator<derived, container>{});

    // const iterators
    test(beman::optional::detail::contiguous_iterator<const int, container>{});
    test(beman::optional::detail::contiguous_iterator<const empty, container>{});
    test(beman::optional::detail::contiguous_iterator<const no_default_ctor, container>{});
    test(beman::optional::detail::contiguous_iterator<const base, container>{});
    test(beman::optional::detail::contiguous_iterator<const derived, container>{});
 ```

I removed this check from the header.